plugins/azure-devops/README.md

```diff
@@ -61,6 +61,25 @@ spec:
   # ...
 ```
 
+#### Mono repos
+
+If you have multiple entities within a single repo, you will need to specify which pipelines belong to each entity.
+
+```yaml
+dev.azure.com/project-repo: <my-project>/<my-repo>
+dev.azure.com/build-definition: <build-definition-name>
+```
+
+#### Pipeline in different project to repo
+
+If your pipeline is in a different project to the source code, you will need to specify this in the project annotation.
+
+```yaml
+dev.azure.com/project-repo: <project1>/<my-repo>
+dev.azure.com/build-definition: <build-definition-name>
+dev.azure.com/project: <project2>
```

Comment:
Can't seem to do a multi-line suggestion but can we use more explicit examples, say like this:

```yaml
dev.azure.com/project-repo: <project-with-source-code>/<my-repo>
dev.azure.com/build-definition: <build-definition-name>
dev.azure.com/project: <project-with-build-code>
```

Feel free to adjust if you have something better, I was just confused by the numbers and feel others might as well.

Reply:
RE: Missed test - Does this not cover it?
plugins/azure-devops/src/plugin.test.ts:72
`it('should be false when no annotations are present', () =>`

Reply:
@awanlin  For a multi line suggestion click on the top line, then shift click the second line before adding your comment.  Then your comment / suggestion should apply to multiple lines

![image](https://github.com/backstage/backstage/assets/289860/f8ef15a2-a4b0-4c2d-97be-8e25b178629c)

Reply:
Changes pushed, hopefully this has everything we need

Reply:
Thanks @afscrome I did try that at the time but it just put the first line, probably user error though, lol